packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -152,28 +152,28 @@ class _CpuProfilerState extends State<CpuProfiler>
     final theme = Theme.of(context);
     final textTheme = theme.textTheme;
     final colorScheme = theme.colorScheme;
-    final currentTab =
-        widget.tabs.isNotEmpty ? widget.tabs[_tabController.index] : null;
-    final hasData =
-        data != CpuProfilerController.baseStateCpuProfileData && !data.isEmpty;
```

Comment:
why did we get rid of this?

Reply:
Because the check at [this line](https://github.com/flutter/devtools/pull/4404/files/f818fffcae967923ef6959b1fe6468276d41749e#diff-9ff54db465627794737ca5328d8d48b29e6586feff364b3ca672c3e22389f058R194) makes it redundant.